### PR TITLE
[2019.2] Fix InputView freeze

### PR DESCRIPTION
**Pull request by Kink3d**
### Purpose of this PR
Fixes an issue where changes to `PortInputView` on Property Nodes freezes the graph. This reproduces in many common operations with Properties. This is a HUP case with ES escalation.

https://fogbugz.unity3d.com/f/cases/1176944/

---
### Release Notes
The FB case linked above is a backport case for the main case (which was marked as fixed during 19.1). This main case has many duiplicates, most of which with different repro steps. I recommend checking a few of these.

---
### Testing status
**Katana Tests**: Running

**Manual Tests**: What did you do?
- [x] Tested UI multi-edition + Undo/Redo
- [x] C# and shader warnings (supress shader cache to see them)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please) N/A

---
### Overall Product Risks
**Technical Risk**: Low - Back/forward port of a bugfix well tested already

**Halo Effect**: None, Low, Medium, High? Low - Tiny bugfix but does affect all default inputs

### CODE REVIEW

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed an error that previously occurred when you used `Sampler State` input ports on Sub Graphs.
 - Fixed various dependency tracking issues with Sub Graphs and HLSL files from Custom Function Nodes.
 - Added support of Alembic velocity to various shaders
+- Renaming Properties no longer freezes the Shader Graph window.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: The Shader Graph window no longer freezes when you rename properties.

**Comment by LandonTownsendUnity** [comment]
Still reproduces


inputviewfreezestillrepros

**Comment by LandonTownsendUnity** [comment]
Confirmed for fixed

**Comment by Kink3d** [comment]
HDRP failures from instability. Merging.

**Pull request merged**